lib/util/versioning.js.orig

```diff
@@ -0,0 +1,317 @@
+"use strict";
```

Comment:
I presume you added this `lib/util/versioning.js.orig` by accident? Can you remove and rebase?